app/scripts/acs/acs-graphs.js

```diff
@@ -51,7 +52,8 @@
                         .x(function(d) { return addElipses(d.label, 30); })
                         .y(function(d) { return d.value; })
                         .tooltipContent(function(d) {
-                            return '<b>' + d.data.label + ': </b>' + $filter('number')(d.data.value);
+                            var percent = $filter('number')(d.data.value / total * 100, 1);
```

Comment:
A couple of small things: since `$filter('number')` is getting referenced four times here, might as well store it as a local variable. For the HTML snippet concatenation, we often write that as a string array with a `.join('')` at the end, for readability.